tests/pytorch/nightly/sd-model.libsonnet

```diff
@@ -39,45 +45,57 @@ local utils = import 'templates/utils.libsonnet';
   tpuVm:: common.PyTorchTpuVmMixin {
     tpuSettings+: {
       tpuVmExports+: |||
-        export XRT_TPU_CONFIG="localservice;0;localhost:51011"
-        export TPU_NUM_DEVICES=4
         cd stable-diffusion/
       |||,
       tpuVmExtraSetup: |||
-        git clone https://github.com/ssusie/stable-diffusion.git
+        git clone https://github.com/gkroiz/stable-diffusion.git
```

Comment:
This currently reads from my fork, but in the long term we should download the lightning stable diffusion test from a more appropriate location. @RissyRan do you have any insight here?

Reply:
What specific changes included in your fork? I would say if changes are for passing the test, we may need to merge them via a PR. If they are making performant, we probably could directly use the public version. Since we still have 5/5 timeline, I am ok to use personal fork for now.

Reply:
My fork is actually based on @ssusie's [fork](https://github.com/ssusie/stable-diffusion). The main difference between @ssusie's fork and the original CompVis repo is support for TPUs. I'm not sure how signifcant the changes are here and whether it is easy to merge into the original CompVis repo. The main difference between my fork and @ssusie's fork is that mine uses the latest Pytorch Lightning version whereas hers uses Pytorch Lightning 1.4.2. I'm slightly hesitant to merge everything into the CompVis repo since we do not own it and it's a research oriented repo so we are unsure what their maintainability in the future looks like.

Reply:
Like you said, we do not need to resolve this before the 5/5 deadline.